macros/edr/tests/test_exposure_schema_validity.sql

```diff
@@ -0,0 +1,58 @@
+{% test exposure_schema_validity(model, exposures, node, columns) %}
```

Comment:
Can you add the new test to `common_test_configs.sql`?
(and maybe also to `get_test_type.sql` though I'm actually not sure it's needed there)

Reply:
needed it in the `get_test_type.sql` (https://github.com/elementary-data/dbt-data-reliability/pull/530/files#diff-7767fb1ef194c47d2d634293a374325976c7d9383d80c7610d8e82637617bf00R33), will add to `common_test_configs.sql` as well

Reply:
ah sorry missed it, cool